src/pages/workspaces/workspace/WorkspaceContainer.test.js

```diff
@@ -0,0 +1,138 @@
+import { render, screen, within } from '@testing-library/react';
+import { axe } from 'jest-axe';
+import { h } from 'react-hyperscript-helpers';
+import { WorkspacePermissionNotice, WorkspaceTabs } from 'src/pages/workspaces/workspace/WorkspaceContainer';
+
+// Mocking for Nav.getLink
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
+  getLink: jest.fn(() => '/'),
+}));
+
+const mockWorkspaceMenu = jest.fn();
+jest.mock(
+  'src/pages/workspaces/workspace/WorkspaceMenu',
+  () =>
+    function (props) {
+      mockWorkspaceMenu(props);
+      return <mock-workspaceMenu />; // eslint-disable-line
```

Comment:
@nawatts @msilva-broad Do either of you happen to know what I can return here to avoid lint warnings about returning a jsx component in a non-jsx file? I couldn't figure out a syntax that worked with the react hyperscript helpers.

Reply:
There's no need to return a React element here. The mocked WorkspaceMenu component can return null.

```suggestion
      return null
```

Reply:
Thanks @nawatts ! I tried returning various things, but never null. :(

I'll fix that up at some point.